setup.py

```diff
@@ -48,6 +48,42 @@
 It should work without additional code in any environment with
 Python >= 2.4, ctypes and an pre-built usb backend library
 (currently, libusb 0.1.x, libusb 1.x, and OpenUSB).
-"""
+""",
+    classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
Next version is supposed to be [release candidate](https://github.com/walac/pyusb/blob/master/usb/__init__.py#L49). Moreover, is there a way to avoid duplicate info with `__init__.py`?